src/main/java/songbox/house/service/impl/GoogleAuthenticationServiceImpl.java

```diff
@@ -159,6 +164,9 @@ private GoogleClientSecrets getClientSecrets() {
     private void saveToken(GoogleTokenResponse googleTokenResponse, Long userId) {
         GoogleApiToken googleApiToken = createToken(googleTokenResponse, userId);
 
+        // Delete
```

Comment:
you can just move next line to a new method and name it `delete(Long userId)` - then comment could be removed.